semgrep/tests/conftest.py

```diff
@@ -101,7 +102,7 @@ def _run_semgrep(
         stderr=subprocess.STDOUT if stderr else subprocess.PIPE,
     )
 
-    if output_format in {"json", "sarif"} and not stderr:
+    if output_format.is_json() and not stderr:
```

Comment:
Is also including JSON_DEBUG here intended?

Reply:
`JSON_DEBUG` is gone after https://github.com/returntocorp/semgrep/pull/3050!

Reply:
<3